src/porepy/numerics/fv/mpfa.py

```diff
@@ -9,6 +9,7 @@
 import scipy.sparse as sps
 
 import porepy as pp
+from porepy.numerics.fv import fvutils
```

Comment:
```suggestion
```

Reply:
This file uses pp.fvutils instead of a direct import of fvutils. While that may be a poor decision (blame the 2016 version of me), we should be consistent within the module.